domains/voxj.json

```
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "voxj",
+           "email": "129677054+voxj@users.noreply.github.com",
+           "discord": "1108039808649015336"
+        },
+    
+        "record": {
+            "CNAME": "https://rawintosh.github.io/"
```

`CNAME` can't contain protocol or `/`
```suggestion
            "CNAME": "rawintosh.github.io"
```